docs/plugins/trello-github-integ_plugin.md

```diff
@@ -1,23 +1,10 @@
 ## 1 `trello-github-integ` Plugin
 
-This plugin creates a new Trello board and integrate it with your GitHub repo.
+This plugin creates a new github actions file(trello-github-integration) and upload to your GitHub repo.
```

Comment:
github actions -> [GitHub Actions](https://github.com/features/actions)

We should use the official name.